rt_gene/src/rt_gene/extract_landmarks_method_base.py

```diff
@@ -7,6 +7,9 @@
 from torch.backends import cudnn as cudnn
 from tqdm import tqdm
 
+from rt_gene.download_tools import download_external_landmark_models
+download_external_landmark_models()
```

Comment:
Note that this cannot be moved in the `__init__` as the SFDDetector import below requires the models to be existing.